components/sandbox/tiers/production/appstudio-env/tiertemplate-appstudio-env-admin-37575401-37575401.yaml

```diff
@@ -12,7 +12,8 @@ spec:
   template:
     apiVersion: template.openshift.io/v1
     kind: Template
-    metadata: {}
+    metadata:
+      creationTimestamp: null
```

Comment:
all those creationTimestamp chages should not be there. This is a bug in ksctl so you need to build a custom version of ksctl including this change: https://github.com/kubesaw/ksctl/pull/50

I can re-generate the tiers for you and update the PR if you want.

Reply:
@hugares  I redid the change after installing the tool from that branch. Please take another look and let me know.

Reply:
looks good, Thanks!